common/build/eslint-plugin-fluid/src/test/mockFiles/no-unchecked-array-access/fileWithArrayAndRecordAccess.ts

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const arr1 = [1, 2, 3];
+const value1 = arr1[0]; // This should not report an error
+
+const record = { a: 1, b: 2 };
+const uncheckedAccess = record["c"]; // This should report an error
```

Comment:
Please use test cases representative of what we are trying to catch.
You want to have test case (ignoring arrays) that are okay without noUncheckedIndexAccess and fail otherwise. This is an error no matter what. This is an example of something that needs to be caught (differs per noUncheckedIndexAccess setting):
```typescript
type SomeObj = { [key: string]: string }
const someObj: SomeObj = { a: "hello", b: "goodbye"};
someObj.a.length;
```
Stick that in [TS playground](https://www.typescriptlang.org/play/) and notice errors when changing noUncheckedIndexAccess within config.